File: Python/items_count.py
# p61
ENDDATA = -1


def items_count():
    array = [54, 20, 88, 30, 71, ENDDATA]
    count = 0
    i = 0

    while array[i] != ENDDATA:
        count += 1
        i += 1

    print("有効データ数 = %d" % count)


items_count()

File: TypeScript/ItemsCount.ts
//p61
const ENDDATA = -1

const ItemsCount = () => {
    const array: number[] = [54, 20, 88, 30, 71, ENDDATA]
    let count: number = 0
    let i = 0

    while (array[i] !== ENDDATA) {
        count++
        i++
    }

    console.log(`有効数データ数 = ${count}`)

}

ItemsCount()